api/dogBreeds/dogBreeds-middleware.js

```diff
@@ -0,0 +1,41 @@
+const db = require(`../../data/db-config`)
+const Breeds = require(`./dogBreeds-model`)
+
+//[POST] + [PUT] requests - checks if req.body has size + breed:
```

Comment:
I know that the requirements of this project wasn't necessarily to have middleware but if you create middleware functions it's good to also create tests for them as well